doc/classes/EditorPlugin.xml

```diff
@@ -748,9 +748,10 @@
 				Emitted when user changes the workspace ([b]2D[/b], [b]3D[/b], [b]Script[/b], [b]AssetLib[/b]). Also works with custom screens defined by plugins.
 			</description>
 		</signal>
-		<signal name="project_settings_changed">
+		<signal name="project_settings_changed" is_deprecated="true">
```

Comment:
I suspected this might be the case. Need to update the schema file to support this attribute here (and ideally update the help to display it too). Perhaps it can be done separately, as a precursor for this change.

Reply:
The help seems to support it from a glance at the `editor_help` code but doesn't have a schema